config.rb

```diff
@@ -71,6 +74,10 @@ def initialize(app, options_hash={}, &block)
 # Helpers
 # ----------------------------------------------
 helpers do
+  # turn string to date object for comparison
+  def d(string)
```

Comment:
>  it depends on the people that they will develop it

yes.  and the people developing this site could be changing a lot.  so we should choose a name that makes send to the next junior developer.

the name should describe what it is doing or what it is used for, so I would suggest something like `parse_date` maybe?